.gitignore

```diff
@@ -13,3 +13,4 @@ pkg/ingester/client/cortex.pb.go
 pkg/ring/ring.pb.go
 bazel-*
 images/
+.idea/
```

Comment:
suspect this wasn't meant to be in the PR?

Reply:
No it was not... removed in latest push